src/Files.App/Views/Shells/ModernShellPage.xaml.cs

```diff
@@ -1,49 +1,57 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using Files.App.Data.EventArguments;
-using Files.App.Extensions;
-using Files.App.Filesystem;
-using Files.App.Helpers;
-using Files.App.UserControls;
-using Files.App.UserControls.MultitaskingControl;
-using Files.App.ViewModels;
-using Files.App.Views.LayoutModes;
-using Files.Backend.Enums;
-using Files.Backend.ViewModels.Dialogs.AddItemDialog;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Input;
 using Microsoft.UI.Xaml.Media.Animation;
 using Microsoft.UI.Xaml.Navigation;
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using Windows.Storage;
 using Windows.System;
 
-namespace Files.App.Views
+namespace Files.App.Views.Shells
 {
 	public sealed partial class ModernShellPage : BaseShellPage
 	{
-		public override bool CanNavigateBackward => ItemDisplayFrame.CanGoBack;
+		public override bool CanNavigateBackward
+			=> ItemDisplayFrame.CanGoBack;
 
-		public override bool CanNavigateForward => ItemDisplayFrame.CanGoForward;
+		public override bool CanNavigateForward
+			=> ItemDisplayFrame.CanGoForward;
 
-		protected override Frame ItemDisplay => ItemDisplayFrame;
+		protected override Frame ItemDisplay
+			=> ItemDisplayFrame;
 
 		private NavigationInteractionTracker _navigationInteractionTracker;
 
+		private NavigationParams _NavParams;
+		public NavigationParams NavParams
```

Comment:
Shouldn't it be lowercase since it's a prive field?

Reply:
Backing field should be UpperCase prefixed with underscore according to the GuidLine.

Reply:
Section 2, Item 2

> Use PascalCase prefixed with _ for properties’ backing fields

Reply:
My bad, thank you for letting me know that